python/cudf/cudf/core/dtypes.py

```diff
@@ -221,14 +221,51 @@ class DecimalDtype(ExtensionDtype):
 
     name = "decimal"
     _metadata = ("precision", "scale")
+    _MAX_PRECISION = np.floor(np.log10(np.iinfo("int64").max))
 
     def __init__(self, precision, scale):
```

Comment:
Should we consider making scale default to 0?

Reply:
Done.